lib/PanelTestProvider.jsm

```diff
@@ -60,6 +60,22 @@ const MESSAGES = () => [
     targeting: `!hasAccessedFxAPanel && !usesFirefoxSync && isFxAEnabled == true`,
     trigger: { id: "toolbarBadgeUpdate" },
   },
+  {
+    id: "WNP_PROFILE_AGE",
+    template: "toolbar_badge",
+    content: {
+      action: {
+        id: "moments-wnp",
+        data: {
+          url: "foo.com",
```

Comment:
Just FYI it's likely we're going to need to run this through the URL formatter like `%LOCALE%`etc

Reply:
Ed is already doing that in his PR https://phabricator.services.mozilla.com/D39286
I could add a check for valid URL and I would update that dummy data with something that matches.